### PR TITLE
v118

### PR DESCRIPTION
**Pull request by Thorin-Oakenpants**

### CODE REVIEW

**Comment by Thorin-Oakenpants** [comment]
open an issue if you want to discuss something

File: user.js [hunk 1]
```diff
@@ -1182,62 +1157,20 @@ user_pref("browser.messaging-system.whatsNewPanel.enabled", false);
...
-   // user_pref("dom.webnotifications.serviceworker.enabled", false);
-// ***/
+/* 9998: deprecated FF103-115 ***/
+   // user_pref("browser.cache.offline.enable", "");
+   // user_pref("extensions.formautofill.heuristics.enabled", "");
+   // user_pref("network.cookie.lifetimePolicy", "");
+   // user_pref("privacy.clearsitedata.cache.enabled", "");
+   // user_pref("privacy.resistFingerprinting.testGranularityMask", "");
+   // user_pref("security.pki.sha1_enforcement_level", "");
+/* 9999: deprecated FF116-128 ***/
```
Comment (nihil-admirari) [inline user.js:1169]: May I request a revert to `ESR115.x still uses all the following prefs` etc? I'm using an ESR with a custom `user.js`, and despite the suggestion to use releases, rebasing on master is much easier that trying to port changes from one release tag to another.

Comment (Thorin-Oakenpants) [reply user.js:1167]: I typed out a huge reply and now I see there's nothing here. The answer is no. ESR users from here on in should use the correct version, as the differences are going to become messy with RFP vs FPP

**Review by icpantsparti2 — COMMENTED** [review]
Hi 👖.  Please can you add a `;` to:

```
   // user_pref("network.trr.bootstrapAddr", "10.0.0.1") // [HIDDEN PREF]
```

Thank you kindly 👍 🐌

**Comment by Thorin-Oakenpants** [comment]
> Thank you kindly 👍 🐌

OMG ... did you just snail me :~p

File: user.js (hunk 1 above)
Comment (Thorin-Oakenpants) [reply user.js]: @nihil-admirari OK, I changed back to the old style, for now, so you're good for another 9 months (and probably forever)

File: user.js [hunk 2]
```diff
@@ -26,15 +27,15 @@
...
-    - use https://github.com/arkenfox/user.js/releases/tag/115.1
+  * Use the arkenfox release that matches your Firefox version
+    - DON'T wait for arkenfox to update Firefox, nothing major changes these days
+  * Each release
+    - run prefsCleaner to reset prefs made inactive, including deprecated (9999)
+  * ESR
+    - It is recommended to not use the updater, or you will get a later version which may cause issues.
+      So you should manually append your overrides (and keep a copy), and manually update when you
+      change ESR releases (arkenfox is already past that release)
+    - If you decide to keep updating, then the onus is on - also see section 9999
```
[the remark this answers is not in the document]
Comment (Thorin-Oakenpants) [reply user.js:38]: I don't see the need to give instructions on how to edit or append

**Pull request merged**